docs/swarm/swarm-tutorial/delete-service.md

```diff
@@ -24,6 +24,7 @@ run your manager node. For example, the tutorial uses a machine named
 
     ```
     $ docker service rm helloworld
+    
```

Comment:
I've been trying to include those as a standard for readability. (And it may not be 100% consistent unfortunately) The problem is when there are tables or other more complicated data in the cli ouput, it clutters the doc to have the output on the very next line after the command.). Will check so it is consistent.